docs/keywords.json

```diff
@@ -7014,224 +7014,267 @@
+      "sky plot",
+      "skyplot",
+      "graphic",
+      "satellites",
+      "elevation",
+      "azimuth",
+      "data",
+      "analysis",
+      "tracking",
       "visualization"
     ],
     "category":"Charts and Graphs",
     "release":"3.13.0"
-  }, 
+  },
   "square-f":{
     "alias":[
-      "3.13.0", 
-      "generic", 
-      "shapes", 
-      "box", 
-      "stop", 
+      "3.13.0",
+      "generic",
+      "shapes",
+      "box",
+      "stop",
       "container"
     ],
     "category":"Generic",
     "release":"3.13.0"
-  }, 
+  },
   "video":{
     "alias":[
-      "3.13.0", 
-      "media", 
-      "camera", 
-      "video camera", 
-      "film camera", 
-      "record", 
-      "clip", 
-      "mp4", 
-      "motion", 
-      "tape", 
-      "watch", 
-      "add video", 
-      "upload video", 
+      "3.13.0",
+      "media",
+      "camera",
+      "video camera",
+      "film camera",
+      "record",
+      "clip",
+      "mp4",
+      "motion",
+      "tape",
+      "watch",
+      "add video",
+      "upload video",
       "record video"
     ],
     "category":"Media",
     "release":"3.13.0"
-  }, 
+  },
   "video-plus":{
     "alias":[
-      "3.13.0", 
-      "media", 
-      "camera", 
-      "video camera", 
-      "film camera", 
-      "record", 
-      "clip", 
-      "mp4", 
-      "motion", 
-      "tape", 
-      "watch", 
-      "add video", 
-      "upload video", 
-      "record video", 
+      "3.13.0",
+      "media",
+      "camera",
+      "video camera",
+      "film camera",
+      "record",
+      "clip",
+      "mp4",
+      "motion",
+      "tape",
+      "watch",
+      "add video",
+      "upload video",
+      "record video",
       "plus"
     ],
     "category":"Media",
     "release":"3.13.0"
-  }, 
+  },
   "sound-low":{
     "alias":[
-      "3.13.0", 
-      "generic", 
-      "volume", 
-      "speakers", 
-      "audio ouput", 
-      "loudness", 
-      "audio input", 
-      "audio enabled", 
-      "change volume", 
-      "lower volume", 
+      "3.13.0",
+      "generic",
+      "volume",
+      "speakers",
+      "audio ouput",
+      "loudness",
+      "audio input",
+      "audio enabled",
+      "change volume",
+      "lower volume",
       "noise"
     ],
     "category":"Generic",
     "release":"3.13.0"
-  }, 
+  },
   "app-update":{
     "alias":[
       "3.13.1",
-      "sync", 
-      "reset", 
-      "app update", 
-      "refresh", 
-      "software download", 
-      "software update", 
+      "sync",
+      "reset",
+      "app update",
+      "refresh",
+      "software download",
+      "software update",
       "software refresh"
     ],
     "category":"Generic",
     "release":"3.13.1"
+  },
+  "webhook":{
+    "alias":[
+      "3.14.0",
+      "web hook",
+      "event",
+      "information",
+      "manage",
+      "create",
+      "http request",
+      "payload",
+      "application",
+      "push"
+    ],
+    "category":"04_Symbols_B",
```

Comment:
Are these real category names?

Reply:
Those are how we have our files labeled — but you're completely right that we usually just have the base name without numbers or letters so ill make that change. Thanks